aws-device-farm/custom-test-envs/android-render-test.yml

```diff
@@ -0,0 +1,123 @@
+version: 0.1
+
+# This flag enables your test to run using Device Farm's Amazon Linux 2 test host. For more information,
+# please see https://docs.aws.amazon.com/devicefarm/latest/developerguide/amazon-linux-2.html
+android_test_host: amazon_linux_2
+
+# Phases represent collections of commands that are executed during your test run on the test host.
+phases:
+
+  # The install phase contains commands for installing dependencies to run your tests.
+  # For your convenience, certain dependencies are preinstalled on the test host. To lean about which
+  # software is included with the host, and how to install additional software, please see:
+  # https://docs.aws.amazon.com/devicefarm/latest/developerguide/amazon-linux-2-supported-software.html
+  install:
+    commands:
+ 
+  # The pre-test phase contains commands for setting up your test environment.
+  pre_test:
+    commands:
+      
+  # The test phase contains commands for running your tests.
+  test:
+    commands:
+      # By default, the following ADB command is used by Device Farm to run your Instrumentation test.
+      # Please refer to Android's documentation for more options on running instrumentation tests with adb:
+      # https://developer.android.com/studio/test/command-line#run-tests-with-adb
+      - echo "Starting the Instrumentation test"
```

Comment:
I would suggest to place all this code in a file and call that file, it will make maintenance easier, I think.